apps/openmw/mwmechanics/levelledlist.hpp

```diff
@@ -80,7 +82,50 @@ namespace MWMechanics
                 return getLevelledItem(ref.getPtr().get<ESM::CreatureLevList>()->mBase, true, failChance);
         }
     }
+}
 
+//Returns list of items in group. You get them by returned value->mId.
+inline const std::vector<ESM::LevelledListBase::LevelItem>& itemsInGroup(const std::string& group)
+{
+    MWWorld::ManualRef ref (MWBase::Environment::get().getWorld()->getStore(), group);
+    const ESM::ItemLevList* levItem = ref.getPtr().get<ESM::ItemLevList>()->mBase;
+    const std::vector<ESM::LevelledListBase::LevelItem>& items = levItem->mList;
+    return items;
+}
+
+//Returns true if item @name is in levelled group @group, false if not.
+inline bool itemInLevelledGroup(const std::string& name, const std::string& group)
+{
+    //Get all items from the group
+    const std::vector<ESM::LevelledListBase::LevelItem>& elements = itemsInGroup(group);
+    //And see if item we're looking for is there.
+    for(std::vector<ESM::LevelledListBase::LevelItem>::const_iterator it = elements.begin(); it != elements.end(); ++it)
+    {
+        if(it->mId == name)
+            return true;
+    }
```

Comment:
Can I rely on checking if item is levelled list by checking first two characters ("l_")?. Also, should I change naming from levelledGroup to levelledList?